microxrce_agent.cpp

```diff
@@ -36,13 +36,55 @@ void showHelp()
     std::cout << "    udp <local_port>" << std::endl;
     std::cout << "    tcp <local_port>" << std::endl;
 #else
-    std::cout << "    serial <device_name>" << std::endl;
-    std::cout << "    pseudo-serial" << std::endl;
+    std::cout << "    serial <device_name> [baudrate=<1200, 2400, 4800, 9600, 19200, 38400, 57600, 115200>]" << std::endl;
+    std::cout << "    pseudo-serial [baudrate=<1200, 2400, 4800, 9600, 19200, 38400, 57600, 115200>]" << std::endl;
     std::cout << "    udp <local_port> [<discovery_port>]" << std::endl;
     std::cout << "    tcp <local_port> [<discovery_port>]" << std::endl;
 #endif
 }
 
+speed_t getBaudRate(const char* baudrate_str)
+{
+    speed_t rv;
+    if (strcmp(baudrate_str, "1200"))
```

Comment:
strcmp returns 0 when both strings are equals, so the if condition is false and the code will not pass when it should

Reply:
Yeah! you are right, thank you!